TAF/testScenarios/integrationTest/UC_mqtt_message_bus/core_data_config.robot

```diff
@@ -84,7 +84,7 @@ Verify MQTT Broker Qos
 MQTT Subscriber Received Event is the Same As Service Log
     ${logs}  Run Process  ${WORK_DIR}/TAF/utils/scripts/${DEPLOY_TYPE}/query-docker-logs.sh core-data ${log_timestamp}
     ...     shell=True  stderr=STDOUT  output_encoding=UTF-8
-    ${correlation_line}  Get Lines Containing String  ${logs.stdout}.encode()  Event Published on message queue
+    ${correlation_line}  Get Lines Containing String  ${logs.stdout}.encode()  Event Published on message bus
```

Comment:
On core data log still `Event received on message queue` not `bus`.

`app=core-data source=subscriber.go:71 msg="Event received on message queue. Topic: edgex/events/device/#, Correlation-id: 138db3af-c68d-492a-9f72-44c34c0cf2e2 "`

Reply:
@cherrycl . I created https://github.com/edgexfoundry/edgex-go/pull/4285 to address the messages and updated this PR accordingly